src/panel/pages/events/components/TimelineRow.tsx

```diff
@@ -122,6 +128,7 @@ export const TimelineRow: FC<
             <TimelineNetworkDuration
               key={`n-${p.elements.length}`}
               state="success"
+              isSelected={e.timestamp === selectedEvent?.timestamp}
```

Comment:
I wonder if we might need a more robust way of matching events than the `timestamp`

Reply:
True - I expected the event to have an `id`.

Reply:
Yeah we probably need to add an `id` at some point. Not sure if it should be covered by this PR though